src/website/app/demos/Dropdown/examples/renderTrigger.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+
+import { createStyledComponent } from '../../../../../library/styles';
+import { Target } from 'react-popper';
+import Dropdown from '../../../../../library/Dropdown';
+import data from '../../Menu/components/menuData';
+import renderPropDescription from '../../shared/renderPropDescription';
+
+export default {
+  id: 'render-trigger',
+  title: 'Custom Rendering - renderTrigger',
+  description: `Use the \`renderTrigger\`
+[render prop](https://reactjs.org/docs/render-props.html) to provide custom
+rendering control of the trigger.
```

Comment:
Is there a good link we could provide with examples of possible triggers here since we don't have a specific component to refer people to?